### PR TITLE
Celo erc20 transfer aggs

### PR DESCRIPTION
**Pull request by tomfutago**

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @tomfutago 

- Partition by block_month ✅
- Dummy legacy models added for new spells ✅

[Sample transfers_celo_erc20_agg_day]()
[Sample transfers_celo_erc20_rolling_day]()

File: models/transfers/celo/erc20/transfers_celo_erc20_agg_day.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  'celo' as blockchain,
+  cast(date_trunc('month', tr.block_time) as date) as block_month,
+  date_trunc('day', tr.block_time) as block_day,
+  tr.wallet_address,
+  tr.token_address,
+  cast(null as varchar(10)) as symbol, -- t.symbol, -- null until tokens_celo_erc20 spell is created
+  sum(tr.amount_raw) as amount_raw,
+  sum(tr.amount_raw / power(10, 18)) as amount --sum(tr.amount_raw / power(10, t.decimals)) as amount
+from {{ ref('transfers_celo_erc20') }} tr
+--left join ref('tokens_celo_erc20') t on t.contract_address = tr.token_address
```
Comment (jeff-dude) [inline models/transfers/celo/erc20/transfers_celo_erc20_agg_day.sql:27]: this is now updated, do you want to finalize this PR?

Comment (tomfutago) [reply models/transfers/celo/erc20/transfers_celo_erc20_agg_day.sql:27]: sure, will do

**Review by jeff-dude — APPROVED** [review]
should be all good 🤝

**Pull request merged**